tests/androidresource/AndroidResourceTest.java

```diff
@@ -0,0 +1,12 @@
+import org.checkerframework.checker.androidresource.qual.res.*;
+import src.R;
+
+class AndroidResourceTest {
+    AndroidResourceTest() {
+    }
+
+    void Test() {
+        @XmlRes int xml_res = R.xml.file_paths;
+//        xml_res = R.string.file_paths;    // Flagged by CF
+    }
+}
```

Comment:
Add the newline at the end of files, for consistency and to get rid of the ugly red icon.

Reply:
This will be resolved in #4

Reply:
Resolved.